flight/Modules/FlightStats/flightstatsmodule.c

```diff
@@ -51,14 +51,13 @@
 // Private variables
 static bool module_enabled;
 static struct pios_thread *flightStatsTaskHandle;
-static FlightStatsSettingsData settings;
+static volatile FlightStatsSettingsData settings;
```

Comment:
I wonder if this won't be a little hard to maintain because it takes care to notice that this should be volatile. Any way to improve that?